shared/common-adapters/markdown/service-decoration.tsx

```diff
@@ -11,6 +14,29 @@ import MaybeMention from '../../chat/conversation/maybe-mention'
 import Text, {StylesTextCrossPlatform} from '../text'
 import {StyleOverride} from '.'
 
+type KeybaseLinkProps = {
+  link: string
+  linkStyle: StylesTextCrossPlatform | undefined
```

Comment:
maybe make these optional with a `?`? or do we want do be explicit when we're not passing styles?